### PR TITLE
More end to end test

### PR DESCRIPTION
**Pull request by RomainGoussault**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,8 +15,9 @@ install:
 # command to run tests
 script:
   - flake8 --max-line-length=120
-  - pytest -v --cov=mplc tests/unit_tests.py
-  - travis_wait 30 pytest -v tests/end_to_end_tests.py
+  - pytest -v --cov=mplc --cov-append tests/unit_tests.py
+  - travis_wait 30 pytest -v --cov=mplc --cov-append tests/ml_perf_end_to_end_tests.py
```
Comment (natct10) [inline .travis.yml:19]: cool

**Review by bowni — APPROVED** [review]
Really cool @RomainGoussault !

File: .travis.yml [hunk 2]
```diff
@@ -15,8 +15,9 @@ install:
 # command to run tests
 script:
   - flake8 --max-line-length=120
-  - pytest -v --cov=mplc tests/unit_tests.py
-  - travis_wait 30 pytest -v tests/end_to_end_tests.py
+  - pytest -v --cov=mplc --cov-append tests/unit_tests.py
```
Comment (natct10) [inline .travis.yml:18]: Even for the first?

Comment (RomainGoussault) [reply .travis.yml]: You are right. I fixed it.

**Review by natct10 — APPROVED** [review]
Testotron 3000

**Pull request merged**